migrations/913-websites-lookup-perms.sql

```diff
@@ -0,0 +1 @@
+UPDATE groups SET rules=CONCAT(rules, ',WebsiteLookup:View') WHERE name IN ('Staff', 'Support Staff', 'Carriers and Operators');
```

Comment:
can you change Staff to: `'Apps:*,Users:Edit,Stats:View,AdminTools:View,AccountLookup:View,AppLookup:View,Lookup:View,Websites:*'`
so we don't have to keep adding new ones.